docs/website/versioned_docs/version-3.0.0/command-reference/add-binding.md

```diff
@@ -2,41 +2,50 @@
 title: odo add binding
 ---
 
+## Description
 The `odo add binding` command can add a link between an operator-backed service and a component. odo uses the Service Binding Operator to create this link. Running this command will make the necessary changes to the Devfile, and once pushed to the cluster, it creates an instance of the `ServiceBinding` resource.
```

Comment:
I know it's not a part of this PR, but I was just reading the rendered doc and found it relevant. It's your call to change it here or separately.
```suggestion
The `odo add binding` command adds a link between an Operator-backed service and a component. odo uses the [Service Binding Operator](https://github.com/redhat-developer/service-binding-operator/) to create this link. Running this command will modify the Devfile, and once pushed (using `odo dev`) to the cluster, it creates an instance of the `ServiceBinding` resource.
```
Specifying `odo dev` in the bracket because we don't have `odo push` anymore.